packages/amplify_datastore/example/ios/amplify_datastore_unit_tests/AmplifyModelSchemaUnitTests.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import XCTest
+import Amplify
+import Combine
```

Comment:
you probably don't need that.

Reply:
Removed import combine, I believe I need XCTest and Amplify though

Reply:
git is not very good at rendering the lines I selected for a comment :( It just adds some more lines for context.